gwpy/timeseries/tests/test_io_gwf_lalframe.py

```diff
@@ -162,9 +166,22 @@ def test_write(tmp_path):
     assert_dict_equal(data, data2, assert_quantity_sub_equal)
 
 
-def test_write_no_ifo(tmp_path):
+@pytest.mark.parametrize(
+    'data',
+    [
+        pytest.param(
+            [],
+            marks=pytest.mark.xfail(
+                raises=RuntimeError,
+                reason='Cannot add an empty series to a frame',
```

Comment:
Please add `strict=True` here so that this cannot be fixed 'accidentally' and have the test still marked as xfail.

```suggestion
                reason='Cannot add an empty series to a frame',
                strict=True,
```

Reply:
Here it is: https://github.com/gwpy/gwpy/pull/1507